src/react/databrowser/objects/__tests__/ObjectDetails.test.jsx

```diff
@@ -0,0 +1,140 @@
+import * as hooks from '../../../utils/hooks';
+import { OBJECT_METADATA, S3_OBJECT, SECOND_S3_OBJECT } from '../../../actions/__tests__/utils/testUtil';
+import ObjectDetails, { InfoWarning } from '../ObjectDetails';
+import { List } from 'immutable';
+import Properties from '../details/Properties';
+import React from 'react';
+import { reduxMount } from '../../../utils/test';
+import router from 'react-router';
+
+describe('ObjectDetails', () => {
+    const MULTIPLE_ITEMS_SELECTED_MESSAGE = 'Mutilple items selected.';
```

Comment:
Can you import these constants from `object/ObjectDetails.jsx`.
You will need to export them at `object/ObjectDetails.jsx`.

Reply:
done